### PR TITLE
[Docs] Governance: Update maintainer guidelines

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Notes for Reviewers**

This PR offers additional insight as to the responsibilities and privileges associated with the maintainer role within the Meshery project.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Aisuko — DISMISSED** [review]
LGTM

**Review by Yashsharma1911 — APPROVED** [review]

**Review by lekaf974 — DISMISSED** [review]

File: GOVERNANCE.md [hunk 1]
```diff
@@ -21,31 +21,61 @@ GitHub organization members are people who have `triage` access to all repos in
...
+
+#### Relinquishment of Role
+
+Maintainers can voluntarily step down from their role at any time.  If you decide to step down, please inform the other maintainers of your decision and explain your reasons.  Before stepping down, please help identify and train your replacement(s), and ensure a seamless transition of responsibilities to prevent any disruption of work. When ready, you or another maintainer will submit a pull request removing your name from the MAINTAINERS.md file and into the list below. Another maintainer will adjust your various permissions / access levels  to various project systems accordingly.
 
 #### Abandonment of Role
 
 Any maintainer who is unresponsive or non-participatory (across the various forms of communication used by the project) for a period of two months forfeits maintainer privileges and resumes a contributor role. While much communication is asynchronous, maintainers are required to attend a minimum of one meeting a month in order facilitate high-fidelity, synchronous communication, which is necessary for the quality collaboration, conveyance of complex topics, and real-time advancement of the project.
 
+##### Remeritus Maintainers
```
Comment (leecalcote) [inline GOVERNANCE.md:77]:
```suggestion
##### Emeritus Maintainers
```

**Review by hamza-mohd — DISMISSED** [review]

**Review by pontusringblom — APPROVED** [review]

**Review by marblom007 — DISMISSED** [review]

**Review by hamza-mohd — APPROVED** [review]
yay!

**Review by winkletinkle — COMMENTED** [review]
I love this community. ❤️

**Pull request merged**